### DISCUSSION TITLE
Creating a CNN Accelerator

### DISCUSSION THREAD

**Discussion by PinthaJane** [gem5/gem5#1357] [Q&A]
I am currently working on implementing a CNN accelerator in gem5, and I am finding it extremely challenging to get started. Despite reading the documentation and watching tutorials, I am struggling to make significant progress.

Could anyone provide guidance or share resources on how to approach building a CNN accelerator in gem5? Specific advice on which components to focus on, example implementations, or any relevant experiences would be greatly appreciated.

**Comment by iliasslasri** [comment]
Hi, I'm also interested in this topic, I have some experience with gem5, do you want to work on it together ?

**Reply by PinthaJane** [reply]
yes I would love to

**Reply by iliasslasri** [reply]
can you contact me @ilqqq on discord?

**Reply by PinthaJane** [reply]
I added you
